index.js

```diff
@@ -192,14 +192,18 @@ module.exports = {
       var output = "describe('" + moduleName + "', function() {\n";
 
       tests.forEach(function(test) {
-        output +=
-          "  it('" + test.name + "', function() {\n" +
-          "    if (!" + test.passed + ") {\n" +
-          "      var error = new chai.AssertionError('" + test.errorMessage + "');\n" +
-          "      error.stack = undefined;" +
-          "      throw error;\n" +
-          "    }\n" +
-          "  });\n";
+        output += "  it('" + test.name + "', function() {\n";
+        if (test.passed) {
+          output +=
+            "    // ESLint passed\n";
```

Comment:
@rwjblue updated the comments to be more generic. feel free to merge. probably not worth a backport, but if you feel bored... 😜